hapi/hedera-protobufs/services/response_code.proto

```diff
@@ -1608,16 +1608,17 @@ enum ResponseCodeEnum {
     * The provided fee exempt key list contains an invalid key.
     */
     INVALID_KEY_IN_FEE_EXEMPT_KEY_LIST = 372;
-    
-    /**
-    * The provided fee schedule key contains an invalid key.
-    */
-    INVALID_FEE_SCHEDULE_KEY = 374;
 
     /**
     * If a fee schedule key is not set when we create a topic
     * we cannot add it on update.
     */
-    FEE_SCHEDULE_KEY_CANNOT_BE_UPDATED = 375;
-    
+    FEE_SCHEDULE_KEY_CANNOT_BE_UPDATED = 373;
+
+    /**
+    * If the topic's custom fees are updated the topic SHOULD have a
+    * fee schedule key
+     */
+    FEE_SCHEDULE_KEY_NOT_SET = 374;
+
```

Comment:
We need to be careful about changing this, since applications could rely on the previous behavior. Do we have approval to add this response cocde?

Reply:
Ah, I see now that this PR would merge to `hip-991-topic-fees`, my mistake